pypeit/core/wavecal/templates.py

```diff
@@ -810,6 +813,35 @@ def main(flg):
         for ii in range(len(iroot)):
             wfile = os.path.join(reid_path, iroot[ii])
             build_template(wfile, slits[ii], lcut, binspec, outroot[ii], lowredux=False)
+    # P200 Triplespec
+    if flg & (2**34):
+        reid_path = os.path.join(resource_filename('pypeit', 'data'), 'arc_lines', 'reid_arxiv')
+        iroot = 'p200_triplespec_MasterWaveCalib.fits'
+        iout = 'p200_triplespec.fits'
+        # Load
+        old_file = os.path.join(reid_path, iroot)
+        par = fits.open(old_file)
+        pyp_spec = par[0].header['PYP_SPEC']
+        spectrograph  = load_spectrograph(pyp_spec)
+        orders = spectrograph.orders
```

Comment:
Not sure why you don't just add the orders to the file that you are reading in or hard wire them here. It seems that this is technically safer than grabbing them from spectrograph which can change. Anyway, it does not matter since it is fixed format. I can live with this.